### ISSUE TITLE
make build-docker-full* fails

### ISSUE THREAD

**Issue by azilly-de** [grafana/grafana#58921]
**What happened**:
When trying to follow [Developer Guide](https://github.com/grafana/grafana/blob/HEAD/contribute/developer-guide.md#build-a-docker-image) to build a docker image, building fails at:

```
$ make build-docker-full

...snip...

(re)installing /go/bin/wire-v0.5.0
go: downloading github.com/google/subcommands v1.0.1
go: downloading golang.org/x/tools v0.0.0-20190422233926-fe54fb35175b
generate code from .cue files
go generate ./pkg/plugins/plugindef
go: downloading github.com/grafana/codejen v0.0.2
go: downloading github.com/deepmap/oapi-codegen v1.10.1
go: downloading github.com/grafana/cuetsy v0.1.1
go: downloading github.com/xlab/treeprint v1.1.0
go: downloading golang.org/x/mod v0.7.0
go: downloading github.com/labstack/echo/v4 v4.9.0
go: downloading github.com/labstack/gommon v0.3.1
go: downloading github.com/valyala/fasttemplate v1.2.1
/grafana/embed.go:9:31: pattern kinds/*/*.cue: no matching files found
pkg/plugins/plugindef/plugindef.go:11: running "go": exit status 1
make: *** [Makefile:69: gen-cue] Error 1
The command '/bin/sh -c make build-go' returned a non-zero code: 2
make: *** [Makefile:157: build-docker-full] Error 2
```

This issues affects tested branches/tags v9.3.0-beta1 and  main BUT current release branch v9.2.5 DOES NOT show this behavior and builds as expected.

**What you expected to happen**:

`make build-docker-full`  creates an image on every branch.

**How to reproduce it (as minimally and precisely as possible)**:

```
$ git clone https://github.com/grafana/grafana.git
$ cd grafana
$ make build-docker-full
```

**Anything else we need to know?**:

**Environment**:
- Grafana version: v9.3.0-beta1 , v9.3.x, main
- Data source type & version: --
- OS Grafana is installed on: docker (20.10.21)
- User OS & Browser: Ubuntu, WSL2(Ubuntu) n.a.
- Grafana plugins: n.a.
- Others: n.a.

**Comment by sgowris2** [comment]
I'm facing the same issue.

**Comment by azilly-de** [comment]
ADDENDUM:

If [PR58101](https://github.com/grafana/grafana/pull/58101) #58101 is pulled in then build runs as expected.

**Comment by mpeaton-ng** [comment]
Same issue here.  I have traced it to the path errors in the embed.go definition, or missing "kind" plugin components.

**Comment by mpeaton-ng** [comment]
This patch should work: 
```diff
diff --git a/Dockerfile b/Dockerfile
index 1032ba60ae..c19eaa9f74 100644
--- a/Dockerfile
+++ b/Dockerfile
@@ -9,7 +9,7 @@ COPY .yarn .yarn
 COPY packages packages
 COPY plugins-bundled plugins-bundled
 
-RUN yarn install
+RUN yarn install 
 
 COPY tsconfig.json .eslintrc .editorconfig .browserslistrc .prettierrc.js babel.config.json .linguirc ./
 COPY public public
@@ -34,6 +34,7 @@ COPY pkg pkg
 COPY scripts scripts
 COPY cue.mod cue.mod
 COPY .bingo .bingo
+COPY kinds kinds
 
 RUN go mod verify
 RUN make build-go
diff --git a/Makefile b/Makefile
index 03a2c40b48..f290b47eb5 100644
--- a/Makefile
+++ b/Makefile
@@ -66,10 +66,7 @@ openapi3-gen: swagger-api-spec ## Generates OpenApi 3 specs from the Swagger 2 a
 ##@ Building
 gen-cue: ## Do all CUE/Thema code generation
 	@echo "generate code from .cue files"
-	go generate ./pkg/plugins/plugindef
-	go generate ./kinds/gen.go
-	go generate ./pkg/framework/coremodel
-	go generate ./public/app/plugins
+	go generate CueSchemaFS
 
 gen-go: $(WIRE) gen-cue
 	@echo "generate go files"```

**Comment by zuchka** [comment]
@mpeaton-ng closing this as I believe #58101 resolves the issue. lmk if we need to reopen.

**Comment by mpeaton-ng** [comment]
Hope so.  The patch I posted worked, this branch did not.  Sorry no time to correspond right now.

From: matt abrams ***@***.***>
Sent: Monday, November 28, 2022 3:52 PM
To: grafana/grafana ***@***.***>
Cc: Eaton, Michael [US] (DS) ***@***.***>; Mention ***@***.***>
Subject: EXT :Re: [grafana/grafana] make build-docker-full* fails (Issue #58921)


@mpeaton-ng<https://github.com/mpeaton-ng> closing this as I believe #58101<https://github.com/grafana/grafana/pull/58101> resolves the issue. lmk if we need to reopen.

**Comment by kminehart** [comment]
@azilly-de This Dockerfile change won't be used in a released version of Grafana until 9.4.0+.

**Comment by simPod** [comment]
I encountered this on 

```$ uname -a
Linux aaa 5.10.0-11-amd64 #1 SMP Debian 5.10.92-2 (2022-02-28) x86_64 GNU/Linux```
